src/libcrun/mount_flags.c

```diff
@@ -431,3 +431,17 @@ libcrun_str2mount_flags (const char *name)
 {
   return libcrun_mount_flag_in_word_set (name, strlen (name));
 }
+
+const struct propagation_flags_s *
+get_mount_flags_from_wordlist(void) {
+  struct propagation_flags_s *flags;
+  size_t numWordlistFlags = sizeof(wordlist) / sizeof(wordlist[0]);
+
+  flags = malloc(sizeof(struct propagation_flags_s) * numWordlistFlags);
```

Comment:
use `xmalloc0` and allocate one extra element:

```
flags = malloc0 (sizeof (struct propagation_flags_s) * (num_wordlist_flags + 1));
```

so we are sure it is NULL terminated